components/TechnologieCarousel.js

```diff
@@ -0,0 +1,48 @@
+import dynamic from "next/dynamic";
+import { CarouselProvider, DotGroup, Slider, Slide } from "pure-react-carousel";
+
+import "pure-react-carousel/dist/react-carousel.es.css";
+import styles from "./Carousel.module.css";
+
+const CarouselKeyboardNavigation = dynamic(
+  () => import("./CarouselKeyboardNavigation"),
+  { ssr: false }
+);
+
+export default function TechnologieCarousel({ slides }) {
```

Comment:
```suggestion
export default function TechnologyCarousel({ slides }) {
```
I would try to always use english naming. Also affects file name.